mechanism/digest/src/main/java/org/wildfly/security/mechanism/digest/DigestUtil.java

```diff
@@ -232,10 +266,13 @@ public static byte[] userRealmPasswordDigest(MessageDigest messageDigest, String
     }
 
     /**
-     * Get array of password chars from TwoWayPassword
+     * Get array of password chars from TwoWayPassword.
      *
-     * @return
-     * @throws SaslException
```

Comment:
The `SaslException` import needs to be removed too (CI currently fails with this error:

```
/home/runner/work/wildfly-elytron/wildfly-elytron/mechanism/digest/src/main/java/org/wildfly/security/mechanism/digest/DigestUtil.java:33:8: Unused import - javax.security.sasl.SaslException. [UnusedImports]
```

Reply:
Sorry about that, that was a stupid oversight on my part. Fixed now

Reply:
No worries, @petrberan! Thanks for updating this!